inspirehep/modules/records/serializers/schemas/base.py

```diff
@@ -49,7 +47,7 @@ def load(self, record):
             pybtex.database.Entity: Pybtex entity
         """
         doc_type, fields = bibtex_type_and_fields(record)
-        texkey = get_value(record, 'texkeys[0]')
+        texkey = record['texkeys'][0]
```

Comment:
Love it: let's not use custom code for operations provided by Python itself.

Reply:
`get_value` is not meant for that, it allows accessing fields deeper inside.
`val = get_value(record, 'foo.bar.baz[3]', default=42)` can be done with
```python
try:
    val = record.get('foo', {'bar': {'baz': [_, _, _, 42]}).get('bar', {'baz':  [_, _, _, 42]})[3]
except KeyError:
    val = 42
```
(or equivalently nested `try...except` blocks) which is a lot more cumbersome.

Reply:
I agree that it was a misuse of `get_value ()`.

Reply:
In the case above though, I agree with @puntonim, the expression it's simple enough to not benefit from `get_value` (with the added side effect of failing early if there's no `texkeys` key, though I'm not 100% sure that's expected).

Reply:
So let's try to have this fixed if necessary and merged tomorrow. @szymonlopaciuk does it make sense to go on and build the `Entry` returned in the `data` tuple or it makes more sense to raise a custom exception?

Reply:
@puntonim IMO it makes more sense to carry on, following @michamos' suggestion, and replacing the texkey with the record ID. We could maybe log such case so that we know there's a problem, but I think it's better to output something over nothing. I'll make a fix tomorrow.